plugins/org.eclipse.elk.conn.gmf/src/org/eclipse/elk/conn/gmf/GmfDiagramLayoutConnector.java

```diff
@@ -885,7 +883,7 @@ protected void processConnections(final LayoutMapping mapping) {
         
         for (ConnectionEditPart connection : mapping.getProperty(CONNECTIONS)) {
             boolean isOppositeEdge = false;
-            EdgeLabelPlacement edgeLabelPlacement = EdgeLabelPlacement.UNDEFINED;
+            Optional<EdgeLabelPlacement> edgeLabelPlacement = Optional.empty();
```

Comment:
Why not use `null` to indicate absence?

Reply:
I prefer to use optionals in such cases to explicitly state that an absent value is allowed and expected.